File: android/app/src/main/java/com/tiagohs/cinema_history/models/tmdb/person/ExternalIds.kt
package com.tiagohs.cinema_history.models.tmdb.person

import com.google.gson.annotations.SerializedName

data class ExternalIds (

	@SerializedName("freebase_id") val freebaseId : String,
	@SerializedName("instagram_id") val instagramId : String,
	@SerializedName("tvrage_id") val tvrageId : Int,
	@SerializedName("twitter_id") val twitterId : String,
	@SerializedName("freebase_mid") val freebaseMid : String,
	@SerializedName("imdb_id") val imdbId : String,
	@SerializedName("facebook_id") val facebookId : String
)

File: ios/HistoryCinema/HistoryCinema/Entities/TMDB/Person/ExternalIds.swift
//
//  ExternalIds.swift
//  popmovies
//
//  Created by Tiago Silva on 19/06/19.
//  Copyright © 2019 Tiago Silva. All rights reserved.
//

import Foundation

struct ExternalIds: BaseModel {
    var id: Int? = UUID().hashValue
    var freebaseId : String?
    var instagramId : String?
    var tvRageId : Int?
    var twitterId : String?
    var freebaseMid : String?
    var imdbId : String?
    var facebookId : String?
    
    enum CodingKeys: String, CodingKey {
        case freebaseId = "freebase_id"
        case instagramId = "instagram_id"
        case tvRageId = "tvrage_id"
        case twitterId = "twitter_id"
        case freebaseMid = "freebase_mid"
        case imdbId = "imdb_id"
        case facebookId = "facebook_id"
    }
}
